extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/RemoteSyncHandler.java

```diff
@@ -98,11 +101,11 @@ private void handleRequest(HttpServerRequest event) {
         } else if (event.method().equals(HttpMethod.DELETE)) {
             handleDelete(event);
         } else if (event.method().equals(HttpMethod.POST)) {
-            if (event.path().equals(DEV)) {
+            if (event.path().endsWith(DEV)) {
```

Comment:
Should we be more specific and check the root path too?

Reply:
Yeah, that is probably fine. I was worried if someone forge a request against another path. But authentication will prevent this.